src/components/Button.js

```diff
@@ -36,16 +36,60 @@ const Button = ({ type='', component: Component='button', variant='', onDisabled
       };
     }
 
+    switch (variant) {
+      case 'primary': {
+        props.appearance = 'primary-action-button';
+        break;
+      }
+      case 'secondary': {
+        props.appearance = 'secondary-action-button'
+        break;
+      }
+      case 'danger': {
+        props.appearance = 'primary-action-button';
+        props.hint = 'danger';
+        break;
+      }
+      case 'anchor': {
+        switch (Component) {
+          case 'a': {
+            // ButtonLink uses <a> element
+            Component = UtrechtButtonLink;
+            break;
+          }
+          case 'button': {
+            // LinkButton link uses <button> element
+            Component = UtrechtLinkButton;
+            break;
+          }
+          default: {
+            // Render `UtrechtButton` or custom component
+            break;
+          }
```

Comment:
this *might* have to default to `case 'button'`, but for now I've worked around it